examples/python/notebooks/EBM Feature Importances.ipynb

```diff
@@ -0,0 +1,435 @@
+   "source": [
+    "Global Explanations are useful for understanding what a model finds important, as well as identifying potential flaws in its decision making or the training data. Let's start by computing and displaying a global explanation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from interpret import show\n",
+    "\n",
+    "ebm_global = ebm.explain_global(name='EBM')\n",
+    "show(ebm_global)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The overall importance for each term is calculated as _the average absolute contribution (score) a term (feature or pair) makes when predicting across the training dataset._ This way of measuring term importance tends to favor terms which, on average, have large impact on predictions for many cases. The overall importance is not a measure of positive/negative -- it is a measure of how important each term is in the scores. For regression, these scores are represented in the same units as the y-axis of the feature graphs. For classification, the scores would be in logits."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Going beyond term importances, because EBMs are additive models we can measure exactly how each term contributes to a prediction. Let's take a look at the graph of the term, `LSTAT`, by selecting it in the drop-down menu.\n",
+    "\n",
+    "![Global Explanation - LSTAT](../assets/importance_notebook_global_lstat.png)\n",
+    "\n",
+    "The way to interpret this is that if a new datapoint came in with `LSTAT` = 5, the model adds about +2.7 to the final prediction. However, for a different datapoint with `LSTAT` = 10, the model would now add approx. -0.47 to the prediction.\n",
+    "\n",
+    "To make individual predictions, the model uses each term graph as a look up table, notes the contribution per term, and sums them together with the learned intercept to make a prediction. In regression, the intercept is the mean target (label) of the training set, and each term adds or subtracts to this mean. In classification, the intercept reflects the base rate of the positive class on a log scale. The gray above and below the graph shows the confidence of the model in that region of the graph."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Local Explanations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see the full breakdown of a prediction on a single sample with Local Explanations. Here's how to compute the prediction breakdown for the first sample in our dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from interpret import show\n",
+    "index = 0\n",
+    "show(ebm.explain_local(X[index:index+1], y[index:index+1]))"
```

Comment:
Just curious: why are you selecting the first row this way? I think using `head` or `iloc` would be the more typical way of getting the first row:

```
show(ebm.explain_local(X.head(1), y.head(1)))
```

Reply:
I'm doing this way because it was suggested that we provide code that could be used for any index, and not only the first one.